### DISCUSSION TITLE
I randomly got blown to smithereens

### DISCUSSION THREAD

**Discussion by HaydenOneAndOnly** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1142] [General]
Bro, what the hell even happened??

I was chopping down a tree and minding my own business when I suddenly get blown to ash by a random, massive explosion. It said I died from a nuclear explosion. When I respawned, I saw a line of smoke in the sky pointing towards where I was killed.

Can nukes just randomly hit you whenever?

![2023-08-13_09 37 06](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/142162668/cc5cde49-51af-498e-87a7-ec357f325027)

**Comment by RagexPrince683** [comment]
Skill issue

**Comment by BruhBronxy** [comment]
Underground gas pocket

**Comment by Arthurp-che** [comment]
ядерный метеорит
